frontend/src/metabase/public/components/EmbedFrame/LogoBadge/LogoBadge.tsx

```diff
@@ -15,6 +15,7 @@ export const LogoBadge = ({ dark }: { dark: boolean }) => {
     <ExternalLink
       className={cx(LogoBadgeStyle.metabaseLink, {
         [LogoBadgeStyle.dark]: dark,
+        [LogoBadgeStyle.light]: !dark,
```

Comment:
Required, so the dark theme can inherit `--mb-color-text-primary` without having to define the value again.